src/olympia/access/acl.py

```diff
@@ -175,6 +175,14 @@ def check_unlisted_addons_reviewer(request):
     return action_allowed(request, amo.permissions.ADDONS_REVIEW_UNLISTED)
 
 
+def check_unlisted_addons_viewer(request):
+    unlisted_perms = [
+        amo.permissions.ADDONS_REVIEW_UNLISTED,
```

Comment:
While I can see how it makes things easier, I'm not sure I like the idea of having a function named `check_unlisted_addons_viewer` check if  you're a viewer... or more than that. It can be confusing, and it's not consistent with the rest (mainly because in current code we don't have a function that checks if you're a viewer, the check is done in various places, which I agree is not ideal).

Reply:
I renamed it to `check_unlisted_addons_viewer_or_reviewer` in eeec85729e.